packages/nextjs-mf/src/default-delegate.ts

```diff
@@ -6,15 +9,29 @@ import {
   WebpackRemoteContainer,
 } from '@module-federation/utilities/src/types/index';
 
+/**
+ * Initializing pure as RemoteVars
+ */
 let pure = {} as RemoteVars;
 try {
-  // @ts-ignore
+  /**
+   * Assigning process.env['REMOTES'] to pure if it exists
+   */
+   // @ts-ignore
   pure = process.env['REMOTES'] || {};
 } catch (e) {
   // not in webpack bundle
 }
```

Comment:
The function 'loadScript' does not handle the case where 'keyOrRuntimeRemoteItem' is neither a string nor a 'RuntimeRemote'. This could lead to unexpected behavior.